addons/point_of_sale/models/pos_order.py

```diff
@@ -278,6 +283,49 @@ def _compute_currency_rate(self):
         for order in self:
             order.currency_rate = self.env['res.currency']._get_conversion_rate(order.company_id.currency_id, order.currency_id, order.company_id, order.date_order)
 
+    @api.depends('lines.is_total_cost_computed')
+    def _compute_is_total_cost_computed(self):
+        for order in self:
+            order.is_total_cost_computed = not False in order.lines.mapped('is_total_cost_computed')
+
+    def _compute_total_cost_in_real_time(self):
+        """
+        Compute the total cost of the order when it's processed by the server. It will compute the total cost of all the lines
+        if it's possible. If a margin of one of the order's lines cannot be computed (because of session_id.update_stock_at_closing),
+        then the margin of said order is not computed (it will be computed when closing the session).
+        """
+        for order in self:
+            lines = order.lines
+            anglo_saxon_lines = lines.filtered(lambda l: l.product_id.type == 'product' and l.product_id.cost_method in ['fifo', 'average'])
```

Comment:
same here for 'anglo_saxon' naming maybe 'storable_not_standard_cost_lines' or a better name :)